src/modules/common/corgi/Activity/Activity.jsx

```diff
@@ -15,15 +15,18 @@ const ActivityPropTypes = {
   sender: CorgiType.sender,
 };
 
+const activityTimeOptions = { largest: 2, units: ['d', 'h'] };
```

Comment:
Sorry, my fault, I was wrong when I told you that you don't need 'm' and 's' units. You need them because user needs be able to see the most recent activity, even seconds

Reply:
No worries. Added minutes and seconds.